test/test_inverse.py

```diff
@@ -619,7 +658,7 @@ def test_KFAC_inverse_from_state_dict(
     inv_kfac = KFACInverseLinearOperator(
         kfac, damping=1e-2, retry_double_precision=False, cache=cache, **kwargs
     )
-    test_vec = random.rand(kfac.shape[1])
+    test_vec = torch.rand(kfac.shape[1])
```

Comment:
Why change to `torch.rand` here?

Reply:
Because this PR changes `KFACInverseLinearOperator` into a `PyTorchLinearOperator` (see [here](https://github.com/f-dangel/curvlinops/blob/3f967823ffb89207a02d24de33480f968561191b/curvlinops/inverse.py#L296)) which will error if one tries multiplying it onto a Numpy array. For that to work, you have to call `.to_scipy()` before.

Reply:
Oh of course lol